paddle/fluid/string/string_helper.cc

```diff
@@ -61,6 +61,19 @@ std::string trim_spaces(const std::string& str) {
   return std::string(p, len);
 }
 
+std::string erase_spaces(const std::string& str) {
```

Comment:
https://stackoverflow.com/questions/83439/remove-spaces-from-stdstring-in-c

Reply:
will fix in next pr